lib/msf/core/exploit/http/client.rb

```diff
@@ -320,22 +322,48 @@ def send_request_raw(opts={}, timeout = 20, disconnect = false)
       r = c.request_raw(opts)
 
       if datastore['HttpTrace']
-        print_line('#' * 20)
-        print_line('# Request:')
-        print_line('#' * 20)
-        print_line(r.to_s)
+        request_color, response_color =
+          datastore['HttpTraceColors'].split('/').map { |color| "%#{color}" }
```

Comment:
This lacks any validation, but it should be safe from crashes. I have a patch to `Rex::Text::Color` that should hopefully address the validation issue in the future, but I'm not blocking this PR on that.

Reply:
Right now, the behavior of this feature allows you to colorize either the request or the response: `red` will colorize the request red, `red/` does the same, and `/blu` or `/blue` will colorize the response blue.